Apps/Database/src/Delegates/DBCommunicationDelegate.cs

```diff
@@ -54,8 +54,7 @@ public DBResult<Communication> GetActive()
             };
             result.Payload = this.dbContext.Communication
                 .OrderByDescending(c => c.CreatedDateTime)
-                .FirstOrDefault(c =>
-                    DateTime.UtcNow >= c.EffectiveDateTime && DateTime.UtcNow <= c.ExpiryDateTime);
+                .FirstOrDefault(c => c.CommunicationTypeCode == CommunicationType.Email || (DateTime.UtcNow >= c.EffectiveDateTime && DateTime.UtcNow <= c.ExpiryDateTime));
```

Comment:
would an expired email not be loaded?

Reply:
per our current requirements, I don't think we use Effective Date Time and Expiry Date Time. They may be used for future requirements.